mypy/test/testsubtypes.py

```diff
@@ -67,10 +67,8 @@ def test_interface_subtyping(self) -> None:
         self.assert_equivalent(self.fx.f, self.fx.f)
         self.assert_not_subtype(self.fx.a, self.fx.f)
 
+    @skip
```

Comment:
Perhaps still leave the TODO item comment here and for the other `@skip`s? I sometimes grep for them from time to time.

Reply:
I am thinking about `skip` as some kind of TODO... I can leave the TODO, but you might want to grep for skips too.